include/clap/ext/draft/preset-load.h

```diff
@@ -26,7 +26,8 @@ typedef struct clap_host_preset_load {
    //
    // [main-thread]
    void(CLAP_ABI *on_error)(const clap_host_t *host,
-                            const char        *uri,
+                            uint32_t           location_kind,
+                            const char        *location,
                             int32_t            os_error,
                             const char        *msg);
 
```

Comment:
There should be a load_key parameter as well I think?

Reply:
Yes.